src/renderers/__tests__/ReactCompositeComponent-test.js

```diff
@@ -504,8 +504,8 @@ describe('ReactCompositeComponent', () => {
 
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(console.error.calls.argsFor(0)[0]).toBe(
-      'Warning: defaultProps was defined as an instance property ' +
-        'on Component. Use a static property to define defaultProps instead.',
+      'Warning: setting defaultProps as an instance property on Component is not supported ' +
```

Comment:
Not that it matters too much, but for future reference, we start warnings with a capital letter. It should be `Setting defaultProps [...]`. We don't consider `Warning:` to be part of the sentence.